presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -737,6 +737,41 @@ Optimizer Properties
     .. warning:: The number of possible join orders scales factorially with the number of relations,
                  so increasing this value can cause serious performance issues.
 
+``optimizer.optimize-case-expression-predicate``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``false``
+
+    When set to true, CASE expression predicate gets simplified into a series of AND/OR clauses.
+    For example::
+
+      SELECT * FROM orders
+       WHERE (CASE
+         WHEN status=0 THEN ‘Pending’
+         WHEN status=1 THEN ‘Complete’
+         WHEN status=2 THEN ‘Returned’
+         ELSE ‘Unknown’
+       END) = ‘Pending’
+
+
+    will get simplified into::
+
+      SELECT * FROM orders
+       WHERE status IS NOT NULL AND status=0;
+
+    If the filter condition was to match the ELSE clause ‘Unknown’, it will get translated into::
+
+      SELECT * FROM orders
+       WHERE (status IS NULL OR (status!=0 AND status!=1 and status !=2));
+
+    The simplification avoids branching and string operations making it more efficient and also allows
+    predicate pushdown to happen avoiding a full table scan. This optimizer is to mainly address queries
+    generated by business intelligence tools like Looker that support human readable labels through CASE statements.
```

Comment:
Can you link https://cloud.google.com/looker/docs/reference/param-field-case here?

Reply:
Done.